.agents/skills/architect/SKILL.md

```
@@ -0,0 +1,18 @@
+---
+name: architect
+description: Software Architect. Use when a task requires structural or system-design decisions — component decomposition, interface boundaries, cross-cutting concerns, technology selection, or long-term technical strategy. Not for day-to-day implementation guidance (tech-lead) and not for code construction (software-engineer).
+canonical_source: .claude/agents/architect.md
+canonical_sha: 14d62cd66bf0e345e395678de810887f236a0bbb
+generator: scripts/compile-runtime-agents.sh
+generator_version: 0.3.0
+classification: generated
+---
+
+## Goal
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)